pages/linux/sam.md

```diff
@@ -0,0 +1,27 @@
+# sam
+
+> AWS Serverless Application Model (SAM) CLI.
+
```

Comment:
Perhaps we could add a new line here:

```
> Homepage: <https://github.com/awslabs/aws-sam-cli>
```

This is the new homepage syntax we defined in #2649.

Reply:
Added. Thanks for the suggestion!